sklearn/metrics/tests/test_ranking.py

```diff
@@ -138,6 +138,12 @@ def test_roc_curve():
     assert_equal(fpr.shape, tpr.shape)
     assert_equal(fpr.shape, thresholds.shape)
 
+    # test partial ROC
+    y_true = np.array([0, 0, 1, 1])
+    y_scores = np.array([0.1,  0,  0.1, 0.01])
+    assert_equal(roc_auc_score(y_true, y_scores, max_fpr=0.3), 0.5)
+    assert_equal(roc_auc_score(y_true, y_true, max_fpr=0.7), 1)
```

Comment:
This varies in both data and max_fpr from the previous line. A better test would consider minimal variations to the parameter and the data and show that the changes affect the metric in the expected way. I'd demonstrate for instance that max_fpr=1 equates to leaving it out, then reduce it.